fusor-ember-cli/app/templates/components/rhci-wizard.hbs

```diff
@@ -1,5 +1,5 @@
 <h1>New {{nameRHCI}} Deployment:  {{name}}</h1>
-<ol class="rhci-steps">
+<ul class="rhci-steps">
```

Comment:
changed from ordered to ordered list.
@rhamilto, I don't think this should affect your CSS but just check to be sure and ACK. Thanks.